tools/resource_analysis.py

```diff
@@ -1,18 +1,27 @@
+# This file does some basic analysis on the intersection of the resources
+# with the highest usage
+
 import pickle
 from resource_map_creator import Resource
 
 resource_map = pickle.load( open("../../data/resource_map.pkl", "rb") )
 
 most_used = set()
 
-for i in range(4):
+for i in range(10):
+    temp = None
     most_used_usage = 0
     for resource in resource_map.keys():
-        if (len(resource_map[resource]) >= most_used_usage) and (not resource in most_used):
-            most_used.add(resource)
+        if (len(resource_map[resource]) >= most_used_usage
+            and not resource in most_used
+        ):
```

Comment:
I suggest formatting/indenting it so that the two if statements here are consistent and it's to read through easier maybe?

Reply:
There are not two if statements. The one in red got deleted. I had to change it such that it was under 80 characters so I just split the equality statement thing onto two lines